documentation/homework/week4/src/stores/todo_actions.js

```diff
@@ -0,0 +1,150 @@
+import { observable, action, runInAction } from 'mobx'
+
+// Setting our localStorage
+// import { locals } from '../utils'
+
+// set the default state of the items Object
+const default_item_state = {
+  description: '',
+  deadline: '',
+}
+
+async function request(url, method, inputToString) {
+  const params = {
+    "body": JSON.stringify(inputToString),
```

Comment:
You don't need to put quotes around object keys!

Reply:
this is just a styling thing because i didn't like the style that some with an other without like the following
```
const params = {
      body: JSON.stringify(inputToString),
      headers: {
        "Content-type": "application/json"
      }
    }
```